workflow/automation/lib/MgmtDB.py

```diff
@@ -441,15 +446,15 @@ def get_runnable_tasks(
                     ),
                     (*allowed_tasks, allowed_rels, offset),
                 ).fetchall()
-                runnable_tasks.extend(
-                    [
-                        (*task, self.get_retries(*task, get_WCT=True))
-                        for task in db_tasks
-                        if self._check_dependancy_met(task, logger)
-                        and "{}__{}".format(*task) not in tasks_waiting_for_updates
-                    ]
-                )
-                offset += 100
+            runnable_tasks.extend(
+                [
+                    (*task, self.get_retries(*task, get_WCT=True))
+                    for task in db_tasks
+                    if self._check_dependancy_met(task, logger)
+                    and "{}__{}".format(*task) not in tasks_waiting_for_updates
```

Comment:
Rather than `'{}__{}'.format(*task)` could just have `'__'.join(task)`

Reply:
Or, alternatively, be a bit more explicit in this loop and unwrap `task` in the for loop.